### DISCUSSION TITLE
so-email missing

### DISCUSSION THREAD

**Discussion by Crash0v3r1de** [Security-Onion-Solutions/securityonion#4233] [Unsupported Versions] [answered]
I have not used seucirytonion since the OG days when you had the desktop environment and all that so maybe I'm doing something wrong here but I have my system installed and working and seeing alerts for traffic but I want to setup email alerts but cannot actually run so-email.

So I logged into the machine via ssh as non root user, tried to run sudo so-email and it's missing, I jump into root user via ssh session and try again and I get the same issue. "command not found"

Is there a new non documented way of configuring email alerts for SecurityOnion 2.3? I was looking in the Web UI also but see nothing.

Also I deployed my SecOnion system with the installer ISO on a fresh machine.

Side note also, absolutely loving the new SecurityOnion!

**Comment by B3DTech** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/email.html?highlight=email
